### PR TITLE
Fix default height for auro-dialog elements

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #71 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

The scope of this update corrects an issue where the default size of a dialog was being incorrectly constrained. This update returns the expectation as specified in the demo that height of a dialog window is dictated by content, with a max height of 80% of the view. 

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Bug

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: test/auro-dialog.test.js [hunk 1]
```diff
@@ -162,23 +162,17 @@ describe('auro-drawer', () => {
...
     await el.updated;
     await sleep(100);
     expect(button.hasAttribute('inert')).to.be.true;
     expect(button.getAttribute('aria-hidden')).to.equal('true');
-  
-    el.open = false;
-    await el.updated;
-    await sleep(100);
-    expect(button.hasAttribute('inert')).to.be.false;
-    expect(button.hasAttribute('aria-hidden')).to.be.false;
```
Comment (geoffrich) [inline test/auro-dialog.test.js:181]: Same comment as other PR -- why was this removed?

**Review by geoffrich — COMMENTED** [review]
Core changes look good, but one comment about the tests. Is there a follow up ticket for allowing devs to customize max-width/height further, as mentioned in the original issue?

**Review by geoffrich — APPROVED** [review]
Changes approved, though my previous question still stands - Is there a follow up ticket for allowing devs to customize max-width/height further, as mentioned in the original issue?

**Comment by blackfalcon** [comment]
> Changes approved, though my previous question still stands - Is there a follow up ticket for allowing devs to customize max-width/height further, as mentioned in the original issue?

- https://github.com/AlaskaAirlines/auro-interruption/issues/77
